.markdownlint.yaml

```
@@ -3,4 +3,4 @@
   code_blocks: false
 MD013: false
 MD024:
-  allow_different_nesting: true
+  siblings_only: true
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)